src/wp-settings.php

```diff
@@ -138,7 +138,7 @@
  *
  * @global string $table_prefix The database table prefix.
  */
-$GLOBALS['table_prefix'] = $table_prefix;
+$GLOBALS['table_prefix'] ??= $table_prefix;
```

Comment:
Please don't use syntax that doesn't work on PHP 7.2. Remove all the other changes and use `isset()` here.